src/runtime_ccall.cpp

```diff
@@ -138,3 +139,17 @@ jl_value_t *jl_get_cpu_name(void)
 #endif
     return jl_pchar_to_string(HostCPUName.data(), HostCPUName.size());
 }
+
+// gh #14591
```

Comment:
I don't think you should add this reference here. If we did that for each code block, it would end up very noisy. `git blame` will give this information when needed.